### DISCUSSION TITLE
vue-tsc RxJS type error

### DISCUSSION THREAD

**Discussion by baronkoko** [vuejs/language-tools#2303] [General]
Hi, all, can anyone help with this error when using RxJS?

![image](https://user-images.githubusercontent.com/27996386/211490379-e38a5a29-5077-435f-bd1a-bb8bdfd3dc72.png)

Reproduction URL:
https://stackblitz.com/edit/vitejs-vite-2fbqrs?file=src%2FApp.vue&terminal=build

Additional context:
In their words, everything is fine on their side ReactiveX/rxjs#7145

**Comment by johnsoncodehk** [comment]
See https://github.com/vuejs/language-tools/issues/1857#issuecomment-1245188245

**Reply by baronkoko** [reply]
thanks @johnsoncodehk!
